src/pages/waiting-room/waiting-room.ts

```diff
@@ -46,12 +48,17 @@ export class WaitingRoomPage extends BasePageComponent {
   ) {
     super(platform, navCtrl, authenticationProvider);
   }
-
   ionViewDidEnter(): void {
     this.store$.dispatch(new waitingRoomActions.WaitingRoomViewDidEnter());
   }
 
   ngOnInit(): void {
+    this.signaturePad.retryImage = '/assets/imgs/waiting-room/retry.png';
```

Comment:
Is there a reason we can't set these within the component ?

Reply:
just trying to add a bit of flexibility for reuse

Reply:
Making it more flexible/reusable is great but I don't see us having a different retry image for each use, maybe having default settings with the option to override them?

There's 2 copies of the same config in the app already , and it's mentioned in the tests. I can see it becoming a pain to change after we add more signature pads further down the line

Reply:
defaulted , with option to override